.terraform.lock.hcl

```diff
@@ -1,132 +1,42 @@
 # This file is maintained automatically by "terraform init".
 # Manual edits may be lost in future updates.
 
-provider "registry.terraform.io/equinix/metal" {
-  version     = "2.1.0"
-  constraints = "~> 2.1"
+provider "registry.terraform.io/equinix/equinix" {
```

Comment:
Since the terraform lock file is ignored by git, please remove the file from the repo rather than updating it.